Jenkinsfile

```diff
@@ -23,6 +30,9 @@ pipeline {
                     git submodule foreach --recursive git reset --hard
                     git submodule foreach --recursive git clean -fdx
                     git submodule status
+                    export commit_sha=$(git submodule status | grep 'pmm-managed' | awk -F ' ' '{print $1}')
+                    curl -s https://api.github.com/repos/percona/pmm-managed/commits/${commit_sha} | grep 'name' | awk -F '"' '{print $4}' | head -1 > OWNER
+                    cat OWNER
```

Comment:
Is it really needed?

Reply:
Yeah, Added for debuggin, forgot to remove...thanks I will update it.